dipy/io/image.py

```diff
@@ -3,13 +3,57 @@
 import numpy as np
 
 
-def load_nifti_data(fname, as_ndarray=True):
+def load_nifti_data(fname, dtype=np.float64):
```

Comment:
Travis will fail here because of `as_ndarray`. This variable is needed, it was the reason for the conflict.

Reply:
Could you please tell me what the docstring should say for this variable?

Reply:
Or feel free to add a docstring in #2031, superseding this PR.

Reply:
Too late it seems, already merged. so you can add: 

```python
as_ndarray: bool
    convert nibabel ArrayProxy to a numpy.ndarray.
    If you want to save memory and delay this operation, just turn this option to False.
    (default: (True))
```